### PR TITLE
Add flags for destroy cluster powervs command

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
**What this PR does / why we need it**:

This PR trying to address this bug https://issues.redhat.com/browse/MULTIARCH-2692 for not handling the flags for destroy cluster powervs command.

**Which issue(s) this PR fixes** *(optional, use `fixes #<issue_number>(, fixes #<issue_number>, ...)` format, where issue_number might be a GitHub issue, or a Jira story*:
Fixes #

**Checklist**
- [x] Subject and description added to both, commit and PR.
- [ ] Relevant issues have been referenced.
- [ ] This change includes docs. 
- [ ] This change includes unit tests.

### CODE REVIEW

File: cmd/cluster/powervs/destroy.go [hunk 1]
```diff
@@ -22,6 +22,14 @@ func NewDestroyCommand(opts *core.DestroyOptions) *cobra.Command {
 		Short:        "Destroys a HostedCluster and its resources on PowerVS",
 		SilenceUsage: true,
 	}
+
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.ResourceGroup, "resource-group", opts.PowerVSPlatform.ResourceGroup, "IBM Cloud Resource group")
+	cmd.Flags().StringVar(&opts.InfraID, "infra-id", opts.InfraID, "Cluster ID with which to tag IBM Cloud resources")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.BaseDomain, "base-domain", opts.PowerVSPlatform.BaseDomain, "Cluster's base domain")
+	cmd.Flags().StringVar(&opts.PowerVSPlatform.Region, "region", opts.PowerVSPlatform.Region, "IBM Cloud region. Default is us-south")
```
Comment (enxebre) [inline cmd/cluster/powervs/destroy.go:29]: why do we need region and VPCRegion?
why do we have 3 different things region, VPCRegion and Zone which has the same default "us-south"?

Comment (dharaneeshvrd) [reply cmd/cluster/powervs/destroy.go:29]: Those are 3 diff fields, only for us-south, all 3 fields sharing the same value.
For osaka, region - osa, zone - osa21 & vpc-region - jp-osa.
https://cluster-api-ibmcloud.sigs.k8s.io/reference/regions-zones-mapping.html

Comment (enxebre) [reply cmd/cluster/powervs/destroy.go:29]: I see, thanks.

You mention default is "us-south", but I can't see that default being set?

Comment (dharaneeshvrd) [reply cmd/cluster/powervs/destroy.go:35]: My bad, added now, thanks!

File: cmd/cluster/powervs/destroy.go [hunk 2]
```diff
@@ -63,13 +77,13 @@ func DestroyCluster(ctx context.Context, o *core.DestroyOptions) error {
 		inputErrors = append(inputErrors, fmt.Errorf("infrastructure ID is required"))
 	}
 	if o.PowerVSPlatform.Region == "" {
```
Comment (enxebre) [inline cmd/cluster/powervs/destroy.go:79]: Upper case seems right to me here even though is an Errorf they are capital letter words regardless.

Comment (dharaneeshvrd) [reply cmd/cluster/powervs/destroy.go]: To be in consistent with other Errorf messages, modified this, reverted the changes.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**